app/helpers/admin/candidates_helper.rb

```diff
@@ -10,4 +10,21 @@ def total_goal
     goals.reduce(:+)
   end
 
+  def campaign_raised_total
+    @current_candidate.candidate_issues.inject(0) {|sum, x| sum + candidate_issue_raised_total(x)}
```

Comment:
Line is too long. [98/80]<br>Space between { and | missing.<br>Space missing inside }.